common/src/main/resources/index-mappings/ml_task.json

```diff
@@ -47,6 +47,9 @@
     "remote_job": {
       "type": "flat_object"
     },
-    "user": USER_MAPPING_PLACEHOLDER
+    "user": USER_MAPPING_PLACEHOLDER,
```

Comment:
Don't forget bump the index version. We met such issue before  https://github.com/opensearch-project/ml-commons/issues/2630, https://github.com/opensearch-project/ml-commons/pull/2710

Reply:
Done